library/ui-styles/src/main/res/color/keyword_foreground_selector.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
+<selector xmlns:android="http://schemas.android.com/apk/res/android">
+    <item  android:alpha="0.25" android:color="?attr/colorOnSecondary" android:state_enabled="false" />
+    <item  android:color="?attr/colorOnSecondary"  />
```

Comment:
I know this is the same value, but since you have used `colorPrimary` for the background, why not using `colorOnPrimary` here (2 occurrences)?

Reply:
Hrmm thought there was a reason for this but no seems you are right. will change, thanks.